agent/util-scripts/pbench-tool-meister-start

```diff
@@ -346,7 +346,14 @@ def main(argv):
     #   - leave a PID file for the tool data sink process
     #   - FIXME: use podman to start a tool-data-sink container
     tds_param_key = "tds-{}".format(group)
-    tds = dict(channel=channel, benchmark_run_dir=benchmark_run_dir)
+    list_hosts = []
```

Comment:
Not sure this construct is necessary, as the Tool Data Sink process has access to all the same information that the Tool Meisters have.  So the Tool Data Sink can load all that information dynamically.